MetaData/python/data13TeV_LFV.py

```diff
@@ -101,6 +101,73 @@ def xsbr(*args, **kwargs):
     'calibrationTarget': 'RunIISpring15DR7',
     'x_sec': 18610.0 ,
 }
+###new stuff
+datadefs["DY1JetsToLL_M-50_TuneCUETP8M1_13TeV-madgraphMLM-pythia8"] = {
+    'analyses': ['4L'],
+    'datasetpath': '/DYJetsToLL_M-10to50_TuneCUETP8M1_13TeV-amcatnloFXFX-pythia8/RunIISpring15DR74-Asympt25ns_MCRUN2_74_V9-v1/MINIAODSIM',
+    'pu': 'Asympt25ns',
+    'calibrationTarget': 'RunIISpring15DR7',
+    'x_sec': 1016 ,
+}
```

Comment:
Hi, I think these (1,2,3,4 jets) are missing the k factor (which I think is k=1.23). The crosssection you have there is the MCM LO one, it wont match to the NNLO reference we usually use for the inclusive samples. 
(The 1.23 comes from the 6025.0 pb NNLO xsec we use for the inclusive sample, divided by the magdraph LO cross section in https://cmsweb.cern.ch/das/request?input=mcm%20prepid=SMP-RunIISummer15GS-00018, 4895 pb )